cmd/fleetctl/scripts.go

```diff
@@ -159,11 +161,15 @@ func runScriptCommand() *cli.Command {
 				return nil
 			}
 
+			s := spinner.New(spinner.CharSets[24], 200*time.Millisecond)
 			if !quiet {
-				fmt.Println("\nScript is running. Please wait for it to finish...")
+				fmt.Println()
+				s.Suffix = " Script is running or will run when the host comes online..."
+				s.Start()
 			}
 
 			res, err := client.RunHostScriptSync(h.ID, b, name, c.Uint("team"))
+			s.Stop()
```

Comment:
So the CLI is stuck with a spinner for up to 1 hour? Any way to cancel and check the results later?

Reply:
I asked this question as well and product is ok for it to be a future enhancement